tagstudio/src/qt/ts_qt.py

```diff
@@ -1187,7 +1203,8 @@ def open_library(self, path: Path) -> LibraryStatus:
         self.filter.page_size = self.lib.prefs(LibraryPrefs.PAGE_SIZE)
 
         # TODO - make this call optional
-        self.add_new_files_callback()
+        if self.lib.entries_count < 10000:
+            self.add_new_files_callback()
```

Comment:
Is this change what the todo refers to? If so can it be removed?

Reply:
I left the `TODO` here since while it originally referred to the persistent call, it still works as a `TODO` to add the option to customize the new threshold here.